recipes/x86_64_level/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set number = 0 %}
+
+package:
+  name: microarch-level-split
+  version: {{ level }}
+
+build:
+  number: {{ number }}
+  string: {{ number }}_{{ microarchitecture }}
+
+outputs:
+  - name: _{{ family }}-microarch-level
+    build:
+      number: {{ number }}
+      string: {{ number }}_{{ microarchitecture }}
+      noarch: generic
+      run_exports:
+        strong:
+          - _{{ family }}-microarch-level >={{ level }}
```

Comment:
I still don't understand why we'd need this here.
But it also doesn't really hurt (apart from confusing me :D ).

Reply:
Yeah, it doesn't hurt to have it there.